L1Trigger/TrackFindingTracklet/python/l1tTTTracksFromTrackletEmulation_cfi.py

```diff
@@ -37,6 +37,6 @@
                                                tableTEDFile = cms.FileInPath('L1Trigger/TrackFindingTracklet/data/table_TED/table_TED_D1PHIA1_D2PHIA1.txt'),
```

Comment:
By default this is off for the track finder, if that is how the track finder have it leave it in their default, if this has been updated in more recent CMSSW releases then rebase to the newer release